artemis.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files -z`.split("\x0").reject {|f| f.match(%r{^(test)/}) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activesupport", ">= 4.2.0"
+  spec.add_dependency "activesupport", ">= 4.0.0"
   spec.add_dependency "graphql", ">= 1.8"
   spec.add_dependency "graphql-client", ">= 0.13.0"
   spec.add_dependency "railties", ">= 4.2.0"
```

Comment:
This requirement also needs to be relaxed.

Reply:
😸 good spot

Reply:
HMM how did that work on my fork!?